packages/kbn-es/src/custom_snapshots.js

```diff
@@ -19,15 +19,15 @@
 
 const { basename } = require('path');
 
-function isVersionFlag(a) {
-  return a.startsWith('--version');
-}
+// function isVersionFlag(a) {
+//   return a.startsWith('--version');
+// }
 
 function getCustomSnapshotUrl() {
   // force use of manually created snapshots until live ones are available
-  if (!process.env.KBN_ES_SNAPSHOT_URL && !process.argv.some(isVersionFlag)) {
-    return 'https://storage.googleapis.com/kibana-ci-tmp-artifacts/{name}-{version}-{os}-x86_64.{ext}';
-  }
+  // if (!process.env.KBN_ES_SNAPSHOT_URL && !process.argv.some(isVersionFlag)) {
+  //   return 'https://storage.googleapis.com/kibana-ci-tmp-artifacts/{name}-{version}-{os}-x86_64.{ext}';
```

Comment:
Can we return undefined here like we are doing in 7.x to make it easier for backports in the future? 

https://github.com/elastic/kibana/blob/7.x/packages/kbn-es/src/custom_snapshots.js#L28-L31

Reply:
d0ff328